import got from "got"

export const getMembers = async () => {
  try {
  } catch (error) {
    throw new Error(error)
  }
}
